runtime/cmd/cmd.go

```diff
@@ -131,13 +124,42 @@ var typeDeclarations = append(
 
 // PrepareChecker prepares and initializes a checker with a given code as a string,
 // and a filename which is used for pretty-printing errors, if any
-func PrepareChecker(program *ast.Program, filename string, must func(error)) (*sema.Checker, func(error)) {
+func PrepareChecker(program *ast.Program, filename string, codes map[string]string, must func(error)) (*sema.Checker, func(error)) {
 	location := runtime.FileLocation(filename)
 	checker, err := sema.NewChecker(
 		program,
 		location,
 		sema.WithPredeclaredValues(valueDeclarations.ToValueDeclarations()),
 		sema.WithPredeclaredTypes(typeDeclarations),
+		sema.WithImportHandler(
+			func(checker *sema.Checker, location ast.Location) (sema.Import, *sema.CheckerError) {
+				stringLocation, ok := location.(ast.StringLocation)
+
+				if !ok {
+					return nil, &sema.CheckerError{
+						Errors: []error{
+							fmt.Errorf("cannot import `%s`. only files are supported", location),
+						},
+					}
+				}
+
+				importChecker, err := checker.EnsureLoaded(
+					location,
+					func() *ast.Program {
+						filename := string(stringLocation)
+						imported, _ := PrepareProgramFromFile(filename, codes)
```

Comment:
Is this an err that's skipped?

Reply:
It looks like it, but the `PrepareProgramFromFile` function does not return an error as it's second return value, but a `must` function. 

This got introduced at some point and I don't like it, because it's not Go idiomatic and confusing. It would be nice to refactor that at some point.

If you look at `PrepareProgramFromFile`, it already checks the error using the must function, then just returns it so that callers have a convenient way of reporting an error through a single function.